packages/flutter/test/widgets/transitions_test.dart

```diff
@@ -243,6 +243,7 @@ void main() {
       textDirection: TextDirection.ltr,
       child: SizeTransition(
         sizeFactor: animation,
+        crossAxisSizeFactor: 2.0,
```

Comment:
This test works to verify that the `crossAxisSizeFactor` is actually being passed but we should also add a test that verifies the intended behavior. Maybe that test could verify the size of the parent before/after the transition when `crossAxisSizeFactor` is set to `1.0`. Before the size would be equal to the size of the child, and after it would be 0.

Reply:
Sure! Nice idea! I added the test in [test: Test the size transition cross axis size](https://github.com/flutter/flutter/pull/134659/commits/ed2bbcfad93dd8c1f947a9e4b723d67e9e0760b2)